package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "style",
+	"version": "1.0.0",
+	"description": "Universal styling/branding using SCSS",
+	"main": "index.js",
```

Comment:
There isn't an `index.js`... (is there?)

Reply:
Oops, you're right